### PR TITLE
Avoid failing rgba API

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
Avoid failing Colors.rgba API on invalid input to avoid app crashes

## Changelog
Avoid failing Colors.rgba API on invalid input to avoid app crashes

### CODE REVIEW

File: src/style/colors.ts [hunk 1]
```diff
@@ -129,7 +130,8 @@ export class Colors {
       blue = validateRGB(p3!);
       opacity = p4;
     } else {
-      throw new Error('rgba can work with either 2 or 4 arguments');
+      LogService.error('Colors.rgba fail due to invalid arguments');
```
Comment (M-i-k-e-l) [inline src/style/colors.ts:133]: I think it should be `failed` (also in tests)

File: src/style/__tests__/colors.spec.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import uut from '../colors';
+import LogService from '../../services/LogService';
+
 const SYSTEM_COLORS = ['grey', 'white', 'black'];
 const GetColorsByHexOptions = {validColors: SYSTEM_COLORS};
 
 describe('style/Colors', () => {
+  const logServiceSpy = jest.spyOn(LogService, 'error');
+  // jest.mock('../../services/LogService');
```
Comment (M-i-k-e-l) [inline src/style/__tests__/colors.spec.js:9]: Can be deleted

File: src/style/__tests__/colors.spec.js [hunk 3]
```diff
@@ -29,7 +33,10 @@ describe('style/Colors', () => {
   });
 
   it('should handle wrong number of params', () => {
-    expect(() => uut.rgba(101, 136, 0.7)).toThrow(new Error('rgba can work with either 2 or 4 arguments'));
+    expect(uut.rgba(101, 136, 0.7)).toBe(undefined);
+    expect(logServiceSpy).toHaveBeenCalledWith('Colors.rgba fail due to invalid arguments');
+    expect(uut.rgba(undefined, 0.2)).toBe(undefined);
+    expect(logServiceSpy).toHaveBeenCalledWith('Colors.rgba fail due to invalid arguments');
```
Comment (M-i-k-e-l) [inline src/style/__tests__/colors.spec.js:39]:
```suggestion
    expect(logServiceSpy).toHaveBeenNthCalledWith(2, 'Colors.rgba fail due to invalid arguments');
```

**Review by M-i-k-e-l — APPROVED** [review]

File: src/style/colors.ts (hunk 1 above)
Comment (ethanshar) [reply src/style/colors.ts:133]: What do you mean should be failed? 
LogService.error doesn't throw an error

File: src/style/__tests__/colors.spec.js (hunk 2 above)
Comment (ethanshar) [reply src/style/__tests__/colors.spec.js:9]: true

**Pull request merged**